src/doc_emit/html_main.re

```diff
@@ -213,7 +219,14 @@ and show_value = (~options as {resolve, _} as options, value) => {
                         {value.descriptor |> get_suffix |> str}
                       </a>
                     </th>
-                    <td> {show_summary(~resolve, value.description)} </td>
+                    <td>
+                      {if (value.deprecated != None) {
```

Comment:
I think it'd be better just to put a `title="This member is deprecated"` on the previous `th`, rather than this span. The strikethrough should be enough visual indicator on its own.

Reply:
@Lemmmmy Was this resolved intentionally or not? Sorry, felt it was probably worth deleting this here.

Reply:
Actually I don't know, let's keep it for now.